cli/cmd/plugin/standalone-cluster/delete.go

```diff
@@ -43,6 +43,13 @@ func init() {
 }
 
 func teardown(cmd *cobra.Command, args []string) error {
+	fmt.Print("\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n")
```

Comment:
@jpmcb can you update this deprecation notice updated to mention the use of `unmanaged-cluster` since it will ship alongside `standalone-cluster`